linodecli/plugins/k8s-alpha.py

```diff
@@ -0,0 +1,286 @@
+"""
+The k8s-alpha plugin includes a means to deploy Kubernetes clusters on Linode
+"""
+import argparse
+import base64
+import sys
+import os
+from subprocess import call as spcall
+import hashlib
+import shutil
+from terminaltables import SingleTable
+
+plugin_name = os.path.basename(__file__)[:-3]
+
+def call(args, context):
+    parser = argparse.ArgumentParser("{}".format(plugin_name), add_help=False)
+    parser.add_argument('command', metavar='COMMAND', nargs='?', type=str,
+                        help="The clusters command to be invoked.")
+    parsed, args = parser.parse_known_args(args)
+
+    commands = {
+        'create': create,
+        'delete': delete
+    }
+    
+    if parsed.command is None or (parsed.command is None and parsed.help):
+        parser.print_help()
+        print_available_commands(commands)
+        sys.exit(0)
+
+    if parsed.command in commands.keys():
+        commands[parsed.command](args, context)
```

Comment:
Personally, I'd rewrite this as 

```python
call_command = commands.get(parsed.command)

if call_command is not None:
    call_command(args, context)
```

Reply:
And exit otherwise. Safe access is always better I agree.

Reply:
It already has this semantics, so I'm going to leave it.